_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -241,139 +241,259 @@ With the information in this type you can clearly identify the Transport object
 
 ### LocalAttributeShareInfo
 
-The Local Attribute Share Info helps to keep track of how the Local Attribute was received/sent, from whom it was received/who sent it, as well as which Local Attribute it was copied from (in case of a shared Repository Attribute). For example, this enables us to track back who we shared a certain Repository Attribute with, so we are able to notify each of them when changing the Repository Attribute.
+The LocalAttribute Share Info helps to keep track of how the LocalAttribute was received/sent, from whom it was received/who sent it, as well as which LocalAttribute it was copied from (in case of a shared Repository Attribute). For example, this enables us to track back who we shared a certain Repository Attribute with, so we are able to notify each of them when changing the Repository Attribute.
 
-| Name             | Type                    | Description                                                                                                                 |
-| ---------------- | ----------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| requestReference | `string`                | The ID of the Local Request the Local Attribute was received in/sent with.                                                  |
-| peer             | `string`                | The Address of the Identity the Local Attribute was received from/shared with.                                              |
-| sourceAttribute  | `string` \| `undefined` | If the Local Attribute is a copy of a Repository Attribute, then this property contains the ID of the Repository Attribute. |
+| Name             | Type                    | Description                                                                                                                |
+| ---------------- | ----------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| requestReference | `string`                | The ID of the Local Request the LocalAttribute was received in/sent with.                                                  |
+| peer             | `string`                | The Address of the Identity the LocalAttribute was received from/shared with.                                              |
+| sourceAttribute  | `string` \| `undefined` | If the LocalAttribute is a copy of a Repository Attribute, then this property contains the ID of the Repository Attribute. |
 
 ## LocalAttributeListener
 
-A LocalAttributeListener is created when you accept an incoming Request with a [`RegisterAttributeListenerRequestItem`]({% link _docs_integrate/requests-and-requestitems.md %}#registerattributelistenerrequestitem). It is used to keep track of which Attribute Listeners currently exist and what they are listening for.
+<!-- TODO -->
```

Comment:
todo?

Reply:
Yeah, here is the problem, the the RegisterAttributeRequestItem documentation is commented out, such that it's a dead link. So should I just keep it without a link (even though the reader would be confused not to find this RequestItem)?

Reply:
my problem here is that you don't actually comment something out, instead here is only a TODO comment :D

Reply:
Yes, I am fine with removing the TODO comment. I just had it as a reminder, that it's a dead link, since the RegisterAttributeListenerRequestItem is already commented out. Removing the TODO comment won't resolve this problem. So my question is, when I remove it know, how to handle the dead link? Should I just remove it?

Reply:
please comment the item itself in